### ISSUE TITLE
Unable to delegate from parent capability.

### ISSUE THREAD

**Issue by tolson4** [digitalbazaar/ezcap#6]
When passing in a (parent)capability to the delegate method, an error is thrown stating that the parent zcap is not found.

```
(node:951934) UnhandledPromiseRejectionWarning: Error: urn:zcap:z19y9dQtmF6W9vdAdtRcQ27p9 not found.
    at /.../node_modules/@digitalbazaar/ezcap/node_modules/jsonld-signatures/lib/documentLoader.js:66:9
    at /.../node_modules/@digitalbazaar/ezcap/node_modules/jsonld-signatures/lib/documentLoader.js:47:12
    at Function.jsonld.get (/.../node_modules/jsonld/lib/jsonld.js:876:27)
    at Function.jsonld.expand (/.../node_modules/jsonld/lib/jsonld.js:309:36)
    at Function.jsonld.compact (/.../node_modules/jsonld/lib/jsonld.js:174:29)
    at Object.api.fetchInSecurityContext (/.../node_modules/ocapld/lib/utils.js:150:17)
    at Object.api.computeCapabilityChain (/.../node_modules/ocapld/lib/utils.js:239:28)
    at CapabilityDelegation.update (/.../node_modules/ocapld/lib/CapabilityDelegation.js:228:37)
    at async Ed25519Signature2018.createProof (/.../node_modules/@digitalbazaar/ezcap/node_modules/jsonld-signatures/lib/suites/LinkedDataSignature.js:107:13)
    at async ProofSet.add (/.../node_modules/@digitalbazaar/ezcap/node_modules/jsonld-signatures/lib/ProofSet.js:109:19)
    at async Proxy.sign (/.../node_modules/@digitalbazaar/ezcap/node_modules/jsonld-signatures/lib/jsonld-signatures.js:55:12)
    at async ZcapClient.delegate (/.../node_modules/@digitalbazaar/ezcap/lib/ZcapClient.js:139:39)

```

**Comment by dlongley** [comment]
This should be fixed by PR #8.

**Comment by dlongley** [comment]
Closing as fixed in 1.0.
